service/microceph.go

```diff
@@ -262,6 +263,25 @@ func (s CephService) Port() int64 {
 	return s.port
 }
 
+// GetVersion gets the installed daemon version of the service, and returns an error if the version is not supported.
+func (s CephService) GetVersion(ctx context.Context) (string, error) {
+	status, err := s.m.Status(ctx)
+	if err != nil && api.StatusErrorCheck(err, http.StatusNotFound) {
```

Comment:
Is this to support the check on older snap versions that have the status endpoint on a different URL?

Reply:
Yeah, basically anything short of microcluster v2 is unsupported.